src/c/generate_c_functions.ml

```diff
@@ -4,6 +4,14 @@
 
 
 let () =
+  
+  (* Include headers necessary for Windows *)
+  if Sys.win32 then
+    (* Setting a WINVER to Vista+ is necessary to get types like AI_ADDRCONFIG *)
+    (print_endline "#define WINVER 0x0600";
+    print_endline "#define _WIN32_WINNT 0x0600";
+    print_endline "#include <ws2tcpip.h>");
+    
```

Comment:
There were some headers we need to pull in to get some APIs (like `AI_ADDRCONFIG`). They were introduced in Vista, so we have to set the `WINVER` and `_WIN32_WINNT` values to `0x0600` (Vista)